gapis/service/path/path.go

```diff
@@ -119,6 +120,7 @@ func (n FramebufferObservation) Parent() Node    { return n.Command }
 func (n Field) Parent() Node                     { return oneOfNode(n.Struct) }
 func (n GlobalState) Parent() Node               { return n.After }
 func (n ImageInfo) Parent() Node                 { return nil }
+func (n ImageView) Parent() Node                 { return nil }
```

Comment:
I am not sure making ImageViews their own type in the API is the right way to go.

Reply:
What do you recommend? In this approach, I need to send something new to the UI

Reply:
I agree. I think it would make sense to create a path called `Resource`:
```
message Resource {
  Capture capture = 1;
  ID id =2;
}
```
We can then use it for and link to any type of resource.